kafka-connect/kafka-connect-events/src/main/java/org/apache/iceberg/connect/events/TableName.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.connect.events;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.Arrays;
+import java.util.List;
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+import org.apache.avro.util.Utf8;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+
+public class TableName implements Element {
```

Comment:
`TableIdentifier` is already an Iceberg type so wanted to avoid that, and `TableInfo` sounds like it should contain more than just a name.

Reply:
would `Identifier` or `FullIdentifier` make sense here?

Reply:
`FullTableName` or `TableIdentity` maybe? Having `Table` in the name would be helpful.

Reply:
`FullTableName` sounds like a good idea

Reply:
I made this change.

Reply:
I'm a little late to the party here, but I like `TableReference` which I don't believe is currently used anywhere.  Also, I think we might want to include the catalog name as well.  That I feel could be helpful with the CommitCompletePayload especially since it can help disambiguate where commits are coming from if there are multiple catalogs.

Reply:
Yes, `TableReference` is great, I made this change. Currently the sink (which hasn't been submitted) only supports a single catalog, but I went ahead and made this change also.

Reply:
I was just thinking that there may be multiple syncs running for different catalogs and if they want something to process those events across multiple topics, it really helps to have the additional context.  Thanks!